### DISCUSSION TITLE
Allow Gyro/Acceleromter Controls

### DISCUSSION THREAD

**Discussion by BanchouBoo** [KeyPuncher/WiinUPro#150] [Ideas]
I'd love to use the gyro in the right joycon for mouse input and the accelerometer in the left as gesture input, will this be a feature and, if so, when can it be expected?
EDIT: https://github.com/mfosse/JoyCon-Driver this project has a gyro-to-mouse-input option for the right joycon, maybe looking at that could give you some insight.

**Comment by KeyPuncher** [comment]
I had done some playing around with the Joy-Con's data being present through as an HID device and found some bytes that I believe correspond to the accelerometer. Switching to using RawInput over DirectInput should allow for parsing of this unused data.

It looks like this other project has even more useful information on the devices, a good place for expanding functionality to also include rumble and LED support.
